nncf/torch/dynamic_graph/patch_pytorch.py

```diff
@@ -20,30 +20,55 @@
 
 from nncf.common.utils.logger import logger
 from nncf.common.utils.os import safe_open
-from nncf.torch.dynamic_graph.trace_functions import CustomTraceFunction
-from nncf.torch.dynamic_graph.trace_functions import ForwardTraceOnly
 from nncf.torch.dynamic_graph.trace_tensor import TracedTensor
 from nncf.torch.dynamic_graph.wrappers import ignore_scope
 from nncf.torch.dynamic_graph.wrappers import wrap_module_call
 from nncf.torch.dynamic_graph.wrappers import wrap_operator
 
 
 class PatchedOperatorInfo:
-    def __init__(self, name: str, custom_trace_fn: CustomTraceFunction = None):
+    def __init__(self, name: str, operator_namespace: object):
```

Comment:
We've set up an entire special enum for namespace targeting, use this as the argument instead of passing an entire namespace object

Reply:
Done